### PR TITLE
CodeFeed StoryBook

### PR DESCRIPTION
**Pull request by rijusougata13**
CodeFeed StoryBook
## Description

added props in codefeed storybook , Now It can be control by user  and also provided a user interface.

## How Has This Been Tested?

with npm start

## Screenshots or GIF (In case of UI changes):
Screenshot from 2021-07-01 12-23-20

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rijusougata13** [comment]
added a extra dependency "Prop-types" for manually update state via props in storybook.

**Review by thisuraseniya — APPROVED** [review]

File: src/config/index.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ export const functions = firebase.functions();
 let firebase_messaging;
```
Comment (shehand) [inline src/config/index.js:31]: Haven't we gitignore this file?

Comment (rijusougata13) [reply src/config/index.js:31]: same issue happened here , this branch was created after merging into main branch ( after config file was gitignored). But still It's appearing in PRs.

Comment (rijusougata13) [reply src/config/index.js:31]: issue solved in #141

**Review by Niweera — APPROVED** [review]

**Pull request merged**